packages/web-app/src/pages/finance.tsx

```diff
@@ -144,31 +113,7 @@ const Finance: React.FC = () => {
           </HeaderContainer>
         }
       >
-        {tokens.length === 0 ? (
-          <PageEmptyState
-            title={t('finance.emptyState.title')}
-            subtitle={htmlIn(t)('finance.emptyState.description')}
-            Illustration={
-              <div className="flex">
-                <IllustrationHuman
-                  {...{
-                    body: 'chart',
-                    expression: 'excited',
-                    hair: 'bun',
-                  }}
-                  {...(isMobile
-                    ? {height: 165, width: 295}
-                    : {height: 225, width: 400})}
-                />
-                <IlluObject object={'wallet'} className="-ml-32" />
-              </div>
-            }
-            buttonLabel={t('finance.emptyState.buttonLabel')}
-            onClick={() => {
-              open('deposit');
-            }}
-          />
-        ) : (
+        {tokens.length !== 0 && (
```

Comment:
mmm what happens if tokens ===0? You have removed this condition right?

Reply:
yes. if there are not tokens Tokens and latest transfers sections won't be shown

Reply:
Ok